rtflib/rtf.py

```diff
@@ -23,10 +23,24 @@
 """
 
 from abc import ABCMeta, abstractmethod
+from collections import namedtuple
```

Comment:
I suggest using `from typing import NamedTuple` since it allows a typed named tuple